### PR TITLE
Coalesce package names for adb/fastboot and gtest/gmock

### PR DESCRIPTION
**Pull request by jbeich**

### CODE REVIEW

File: rules.yaml [hunk 1]
```diff
@@ -97,9 +99,12 @@
 - { name: xpi-adblock_plus,            setname: adblock-plus,             repos: [ freebsd ] }
 - { name: xpi-foxyproxy,               setname: foxyproxy,                repos: [ freebsd ] }
 - { name: zookeeper-c,                 setname: zookeeper,                repos: [ gentoo ]  }
+- { namepat: "android-platform-system-(core|extras)", setname: android-tools, repos: [ debian ] }
```
Comment (AMDmi3) [inline rules.yaml]: Is this really valid while there's already android-tools package in Debian?

File: rules.yaml [hunk 2]
```diff
@@ -97,9 +99,12 @@
...
+- { namepat: "android-tools-[^-]+",    setname: android-tools,            repos: [ freebsd, pkgsrc ] }
```
Comment (AMDmi3) [inline rules.yaml]: This only applies to freebsd, no?

File: rules.yaml [hunk 3]
```diff
@@ -97,9 +99,12 @@
...
 - { namepat: "celestia-.*",            setname: celestia,                 repos: [ freebsd ] } # merges glut/gnome/gtk, just GUI variants
 - { namepat: "eclipse-plugin-(.*)",    setname: "eclipse-$1",             repos: [ openbsd ] }
 - { namepat: "eiskaltdcpp-.*",         setname: eiskaltdcpp,              repos: [ freebsd ] }
+- { namepat: "g(oogle)?-?mock",        setname: gmock,                    repos: [ freebsd, debian ] }
```
Comment (AMDmi3) [inline rules.yaml]: Better to just split it to two simple ```name:``` rules for freebsd and debian.

File: rules.yaml (hunk 2 above)
Comment (jbeich) [inline rules.yaml]: pkgsrc-wip also has [android-tools-adb](http://pkgsrc.se/wip/android-tools-adb).

File: rules.yaml (hunk 1 above)
Comment (jbeich) [inline rules.yaml]: adb is part of [platform/system/core](https://android.googlesource.com/platform/system/core/). Debian changed how they package other stuff from that repo, so android-tools is [deprecated](https://wiki.debian.org/AndroidTools/#Original_android-tools_package).

File: rules.yaml (hunk 3 above)
Comment (jbeich) [inline rules.yaml]: OK. Rebased.

File: rules.yaml (hunk 1 above)
Comment (jbeich) [inline rules.yaml]: Mapping Android stuff is going to be a mess either way:
- fastboot (like adb) lives in `core` repo but has dependencies in `extras`, `libselinux`.
- simpleperf lives in `extras` but has dependencies in `core`, `bionic` and `lzma` (in trunk).
- Fedora (and RPM-friends), Slackware, Gentoo, Arch have adb/fastboot in android-tools
- Arch and Debian also have adb/fastboot in *-platform-tools with Android SDK versioning scheme

**Pull request merged**